base/applications/calc/htmlhelp.c

```diff
@@ -49,7 +49,7 @@ dummy_HtmlHelpW(HWND hWnd, LPCWSTR pszFile, UINT uCommand, DWORD dwData)
 
 void HtmlHelp_Start(HINSTANCE hInstance)
 {
-    hHtmlHelp = LoadLibrary(_T("HTMLHELP"));
+    hHtmlHelp = LoadLibrary(_T("HHCTRL.OCX"));
```

Comment:
Why?

Reply:
Please find where htmlhelp.dll exists.
Doing "htmlhelp" with `LoadLibrary()` was my mistake, sorry.

Reply:
Ah yes you are correct.
Also, I see that basically you are trying to implement the "htmlhelp.lib" helper library there, you may consider doing something similar to
https://stackoverflow.com/questions/1748592/loadlibrary-on-ocx-file-fails-in-windows-7-x64
because the path to the hhctrl.ocx installed on the system is especially mentioned in the CLSID Inprocserver32 key value (see the example).

Reply:
I do not see a good reason for polling CLSID and get the full path.
HHCTRL.OCX stays in a known place so just doing `LoadLibrary(T("HHCTRL.OCX"))` is more than enough.
That code at stackoverflow was wrong anyways, because it reads a string with environment variables inside and used it directly.

Reply:
> That code at stackoverflow was wrong anyways, because it reads a string with environment variables inside and used it directly.

Yes correct what was missing was an ExpandEnvironmentStrings call.
But my point was about the CLSID thing.

Reply:
> HHCTRL.OCX stays in a known place so just doing `LoadLibrary(T("HHCTRL.OCX"))` is more than enough.